README.md

```diff
@@ -49,16 +49,19 @@ To add breaks in the video, simply add a new time (in seconds) in the list of br
     });
 
 ### Customize marker style:
-The style of the markers could be modified by passing an optional setting "markerStyle" with your preference of css styles.
+The style of the markers could be modified by passing an optional setting "markerTip.style" with your preference of css styles.
+You can also customise each marker style on it's own.
 
     video.markers({
-      markerStyle: {
-         'width':'8px',
-         'background-color': 'red'
```

Comment:
I'm inclined to not make this change since it is a big breaking change to the API. Moving `markerStyle` to `markerTip.style` isn't appropriate either because markerTip is used for the hover-over pointer thing.